File-Formats/DocIO/convert-markdown-to-word-document-in-csharp.md

```diff
@@ -0,0 +1,575 @@
+    if (args.Uri == "Image_1.png")
+    {
+        Assembly assembly = typeof(App).GetTypeInfo().Assembly;
+        args.ImageStream = assembly.GetManifestResourceStream("Customize_image.Assets.Image_1.png");
+    }
+    else if (args.Uri == "Image_2.png") 
+    {
+        Assembly assembly = typeof(App).GetTypeInfo().Assembly;
+        args.ImageStream = assembly.GetManifestResourceStream("Customize_image.Assets.Image_2.png");
+    }
+    //Retrive the image from the website and use it.
+    else if (args.Uri.StartsWith("https://"))
+    {
+        WebClient client = new WebClient();
+        //Download the image as a stream.
+        byte[] image = client.DownloadData(args.Uri);
+        Stream stream = new MemoryStream(image);
+        //Set the retrieved image from the input Markdown.
+        args.ImageStream = stream;
+    }
+}
+{% endhighlight %}
+
+{% highlight c# tabtitle="ASP.NET Core" %}
+private static void MdImportSettings_ImageNodeVisited(object sender, Syncfusion.Office.Markdown.MdImageNodeVisitedEventArgs args)
+{
+    //Set the image stream based on the image name from the input Markdown.
+    if (args.Uri == "Image_1.png")
+        args.ImageStream = new FileStream("Image_1.png", FileMode.Open);
+    else if (args.Uri == "Image_2.png")
+        args.ImageStream = new FileStream("Image_2.png", FileMode.Open);
+    //Retrive the image from the website and use it.
+    else if (args.Uri.StartsWith("https://"))
+    {
+        WebClient client = new WebClient();
+        //Download the image as a stream.
+        byte[] image = client.DownloadData(args.Uri);
+        Stream stream = new MemoryStream(image);
+        //Set the retrieved image from the input Markdown.
+        args.ImageStream = stream;
+    }
+}
+{% endhighlight %}
+
+{% highlight c# tabtitle="Xamarin" %}
+private static void MdImportSettings_ImageNodeVisited(object sender, Syncfusion.Office.Markdown.MdImageNodeVisitedEventArgs args)
+{
+    //Set the image stream based on the image name from the input Markdown.
+    if (args.Uri == "Image_1.png")
+    {
+        Assembly assembly = typeof(App).GetTypeInfo().Assembly;
+        args.ImageStream = assembly.GetManifestResourceStream("Customize_image.Assets.Image_1.png");
+    }
+    else if (args.Uri == "Image_2.png") 
+    {
+        Assembly assembly = typeof(App).GetTypeInfo().Assembly;
+        args.ImageStream = assembly.GetManifestResourceStream("Customize_image.Assets.Image_2.png");
+    }
+    //Retrive the image from the website and use it.
+    else if (args.Uri.StartsWith("https://"))
+    {
+        WebClient client = new WebClient();
+        //Download the image as a stream.
+        byte[] image = client.DownloadData(args.Uri);
+        Stream stream = new MemoryStream(image);
+        //Set the retrieved image from the input Markdown.
+        args.ImageStream = stream;
+    }
+}
+{% endhighlight %}
+{% endtabs %}
+
+N> Hook the event handler before opening Word document as per above code example.
```

Comment:
N> Hook the event handler before opening a Word document as per the above code example.